src/views/product-root-docs-path-landing/components/marketing-content/index.tsx

```diff
@@ -137,6 +138,10 @@ const ProductRootDocsPathLandingMarketingContent = ({ blocks }) => {
 	return (
 		<div className={s.root}>
 			{blocks.map((block) => {
+				if (block.type === 'paragraph') {
```

Comment:
New block type.

I'd love to add some validation for the incoming blocks at some point - see 🎟️ [[docs] validate docs custom landing content](https://app.asana.com/0/1202097197789424/1202623957639540/f) - but that felt outside the scope of this PR.